modules/currency-l0/src/main/scala/org/tessellation/currency/l0/http/routes/CurrencyMessageRoutes.scala

```diff
@@ -31,7 +31,8 @@ class CurrencyMessageRoutes[F[_]: Async: Hasher](
   mkCell: CurrencySnapshotEvent => Cell[F, StackF, _, Either[CellError, Ω], _],
   validator: CurrencyMessageValidator[F],
   snapshotStorage: SnapshotStorage[F, CurrencyIncrementalSnapshot, CurrencySnapshotInfo],
-  identifierStorage: IdentifierStorage[F]
+  identifierStorage: IdentifierStorage[F],
+  feesAddressesStorage: FeesAddressesStorage[F]
```

Comment:
I'd be more generic for the storage name. `AddressStorage` or `FeeStorage` 🤔 
`FeesAddresses` is very very specific